server/src/main/java/org/tctalent/server/api/admin/SystemAdminApi.java

```diff
@@ -996,7 +1282,11 @@ public String migrateSurvey() {
 
             Connection targetConn = DriverManager.getConnection(targetJdbcUrl, targetUser, targetPwd);
 
-            log.info("Migration survey data for candidates");
+            LogBuilder.builder(log)
+                .user(authService.getLoggedInUser())
+                .action("MigrateSurvey")
+                .message("Migration survey data for candidates")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We found 15 occurrences of messages that are similar to the selected lines. However, the message in the selected lines is not consistent with the other messages. We recommend you keep the messages in your code consistent. The following are some examples of similar messages in your code: 
talentcatalog/server/src/main/java/org/tctalent/server/api/admin/SystemAdminApi.java#L1840:             .message(&#34;Migration data for candidate educations&#34;)

talentcatalog/server/src/main/java/org/tctalent/server/api/admin/SystemAdminApi.java#L1910:             .message(&#34;Migration data for candidate occupations&#34;)

Reply:
These are logging messages for different application scenarios; therefore, they are different.